.prettierignore

```diff
@@ -1 +1,2 @@
 modules/**
```

Comment:
This and other formatting/linting-related changes were needed to fix the failing CI job. The problem was conflicting eslint-prettier-plugin rules and prettier itself due to misconfiguration.

TLDR: For linting and formatting javascript/typescript code, we use [eslint-plugin-prettier](https://github.com/prettier/eslint-plugin-prettier), which runs prettier as ESLint rules. For checking formatting, we use a standalone `prettier` package instead. After updating dependencies, they introduced conflicting rules. So, ESLint would say that prettier's formatting needs to be corrected and vice versa. To fix this, I updated the prettier script to only format solidity files in the 4337 and added the `examples` directory to `.prettierignore` for the global check (since the only app there uses eslint for formatting as well)